### PR TITLE
Fixes are fixes.. right?

### PR DESCRIPTION
**Pull request by upayanmazumder**
#### Z-index issue

image

#### Previous Design

Old Button Design

#### New Design

New Button Design

### CODE REVIEW

File: pages/shared/footer.css [hunk 1]
```diff
@@ -116,6 +116,7 @@
     grid-auto-flow: column;
     grid-auto-columns: max-content;
     gap: 1rem;
+    z-index: 10;
```
Comment (lucsoft) [inline pages/shared/footer.css:119]: move this to the .icon-bar

File: pages/shared/footer.ts [hunk 2]
```diff
@@ -62,7 +61,6 @@ export function Footer() {
                             ["youtube", "Youtube", "https://www.youtube.com/@bbn6775"],
                             ["twitch", "Twitch", "https://twitch.tv/gd_bbn"],
                             ["twitter", "Twiter", "https://twitter.com/BBN_Holding"],
-                            ["facebook", "Facebook", "https://www.facebook.com/bbn.holding/"],
```
Comment (lucsoft) [inline pages/shared/footer.ts:65]: instead of removing can you replace it with the Mastodon page?

**Review by lucsoft — CHANGES_REQUESTED** [review]

File: pages/shared/footer.ts (hunk 2 above)
Comment (upayanmazumder) [reply pages/shared/footer.ts:65]: what mastodon page bruh

Comment (upayanmazumder) [reply pages/shared/footer.ts:65]: image
bro created it a week back hmm

**Comment by upayanmazumder** [comment]
> NOTE TO SELF : First finalise your commits

**Comment by upayanmazumder** [comment]
merge..?

**Comment by greg6775** [comment]
there's no need to remove the dead links at this point. solution for this would be to start the blog container again etc

**Comment by greg6775** [comment]
furthermore, we're already working on revising all legal pages. the modified code has already been removed in the updated version (not on prod yet)

**Comment by upayanmazumder** [comment]
okkk

**Pull request closed without merging**